lib/certificate-stack.ts

```diff
@@ -0,0 +1,70 @@
+import * as cdk from '@aws-cdk/core';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as sns from '@aws-cdk/aws-sns';
+import * as codebuild from '@aws-cdk/aws-codebuild';
+import * as iam from '@aws-cdk/aws-iam';
+import * as route53 from '@aws-cdk/aws-route53';
+
+export interface CertificateProps extends cdk.NestedStackProps {
+    readonly vpc: ec2.IVpc;
+    readonly notifyTopic: sns.ITopic;
```

Comment:
let’s send the failure of build of certificate project to this topic

Reply:
Done.